### DISCUSSION TITLE
Embedded Dynamic Routes?

### DISCUSSION THREAD

**Discussion by bobbymannino** [sveltejs/kit#5797] [Q&A] [answered]
--- index.svelte
--- sitemap.xml.ts
--- [country]
--- --- index.svelte
--- --- index.ts
--- --- [town]
--- --- --- index.svelte
--- --- --- index.ts
--- --- --- [street]
--- --- --- --- index.svelte
--- --- --- --- index.ts

'[country]' and '[town]' and '[street]' are folders, the rest are files. The town folder seems to be throwing the 404 error. say my URL is 'localhost:1234/england/cornwall' or 'localhost:1234/england/cornwall/launceston' it is throwing an error. Not sure how do do it properly and can't find an answer online, any help would be massively appreciated

**Comment by koodeau** [comment, marked as answer]
`const { country, town } = params;`

Then in fetch use as ...${country}/${town}

You can find some examples [here](https://kit.svelte.dev/docs/routing#pages)
